lib/ocpp/v201/monitoring_updater.cpp

```diff
@@ -0,0 +1,532 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
+
+#include <ocpp/v201/monitoring_updater.hpp>
+
+#include <chrono>
+
+#include <ocpp/v201/ctrlr_component_variables.hpp>
+#include <ocpp/v201/device_model.hpp>
+#include <ocpp/v201/utils.hpp>
+
+namespace ocpp::v201 {
+
+template <DataEnum T>
+bool triggers_monitor(const VariableMonitoringMeta& monitor_meta, const std::string& value_old,
+                      const std::string& value_new) {
+    if constexpr (T == DataEnum::boolean) {
+        return (value_old != value_new);
+    } else {
```

Comment:
You dont need the `else` here since the statement in the if returns

Reply:
it is a 'constexpr' expression, the else is required.